src/js/edu-benefits/1990-rjsf/config/form.js

```diff
@@ -422,53 +433,95 @@ const formConfig = {
     personalInformation: {
       title: 'Personal Information',
       pages: {
-        contactInformation: {
-          title: 'Contact information',
-          path: 'personal-information/contact-information',
+        contactInformation: _.merge(contactInformationPage(fullSchema1990), {
           uiSchema: {
-          },
-          schema: {
-            type: 'object',
-            properties: {
-            }
+            'ui:title': 'Contact information'
           }
-        },
+        }),
         secondaryContact: {
           title: 'Secondary contact',
           path: 'personal-information/secondary-contact',
           uiSchema: {
+            'ui:title': 'Secondary contact',
+            'ui:description': 'This person should know where you can be reached at all times.',
+            secondaryContact: {
+              fullName: {
+                'ui:title': 'Name'
+              },
+              phone: phoneUI('Telephone number'),
+              'view:address': {
+                'ui:title': 'Address'
+              },
+              sameAddress: {
+                'ui:title': 'Address for secondary contact is the same as mine'
+              },
+              address: _.merge(addressUI('', false), {
+                'ui:options': {
+                  hideIf: formData => formData.secondaryContact && formData.secondaryContact.sameAddress
+                }
+              })
+            }
           },
           schema: {
             type: 'object',
             properties: {
+              secondaryContact: {
+                type: 'object',
+                properties: {
+                  fullName: secondaryContact.properties.fullName,
+                  phone,
+                  'view:address': {
+                    type: 'object',
+                    properties: {}
+                  },
+                  sameAddress: secondaryContact.properties.sameAddress,
```

Comment:
I think you could put sameAddress and address inside 'view:address' and you would avoid having the empty object.